### PR TITLE
remove PixelCPEInitial_cfi and PixelCPEParmError_cfi and all their occurences

### PR DESCRIPTION
**Pull request by mmusich**
#### PR description:

This PR removes two ancient pieces of configuration `RecoLocalTracker/SiPixelRecHits/python/PixelCPEInitial_cfi.py` and `RecoLocalTracker/SiPixelRecHits/python/PixelCPEParmError_cfi.py` supposed to support Pixel CPE `ESproducer`s that do not even exist anymore in `cmssw`. All occurrences in other subsystems are removed as well.

#### PR validation:

Passes `runTheMatrix.py -l limited --ibeos`
Zero regressions expected.

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Not a backport, no backport is needed.
cc: @tvami @tsusa

### CODE REVIEW

**Comment by jfernan2** [comment]
please test

File: Alignment/LaserAlignment/python/LaserSeedGenerator_cff.py [hunk 1]
```diff
@@ -3,7 +3,6 @@
 # stripCPE
 from RecoLocalTracker.SiStripRecHitConverter.StripCPEfromTrackAngle_cfi import *
 # pixelCPE
```
Comment (perrotta) [inline Alignment/LaserAlignment/python/LaserSeedGenerator_cff.py:5]: Please also remove the corresponding comment line "`# pixelCPE`" (16 occurencies in this PR)

**Comment by mmusich** [comment]
@cmsbuild please test

**Comment by pohsun** [comment]
+1

**Comment by Martin-Grunewald** [comment]
+1

**Comment by perrotta** [comment]
+1
- Removed unused PixelCPEInitial_cfi and PixelCPEParmError_cfi, as well as all their inclusions in other configs
- Jenkins tests pass and show no differences in reco outputs

**Comment by jfernan2** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Pull request merged**